HoloPipelines/core/wrappers/simplify.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import subprocess
+
+from config import SIMPLIFICATION_RATIO
+
+current_script_directory = (
```

Comment:
Can you move the binary to that subdirectory and create a small README in the directory too please, stating where it is taken from. Like [>this<](https://github.com/nbckr/HoloRepository-Core/blob/PIPELINE/major-refactoring/HoloPipelines/core/third_party/ct_lung_segmentation/README.md).

Reply:
moved and ```README.md``` added